modules/adding-a-model-server-to-your-data-science-project.adoc

```diff
@@ -55,7 +55,7 @@ The *Add model server* dialog opens.
 If you are using a _custom_ model-serving runtime with your model server, you must ensure that your custom runtime supports GPUs and is appropriately configured to use them.
 ====
 . Optional: From the *Accelerator* list, select an accelerator. 
-. Optional: If you selected an accelerator in the preceding step, from the *Number of accelerators* list, select a value.  
+.. If you selected an accelerator in the preceding step, select the number of accelerators to use.
```

Comment:
I verified that it's not a list, but a kind of combo box with a text field and plus and minus buttons to change the value. I'm not sure of the exact name for that type of UI control.

Regardless, I don't think we need to tie ourselves up in knots describing it. I think the following will suffice:

"If you selected an accelerator in the preceding step, specify the number of accelerators to use." (I changed 'select' to 'specify' because the UI control allows you to directly type a value).